### PR TITLE
scalafmt align.openParen false, #26511

### PR DESCRIPTION
**Pull request by patriknw**
```
align.openParenDefnSite    = false
align.openParenCallSite    = false
```

Refs #26511

(sorry that I missed this in previous PR)

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -2,15 +2,14 @@ version = 2.0.0-RC5
 
 style = defaultWithAlign
 
-danglingParentheses        = true
```
Comment (patriknw) [inline .scalafmt.conf]: there is a duplicate below

**Review by johanandren — APPROVED** [review]
Yes, that was very annoying. LGTM!

**Review by raboof — APPROVED** [review]
Just rebased, let's merge sooner rather than later ;)

**Comment by patriknw** [comment]
rebased again, will merge after validation

File: .scalafmt.conf [hunk 2]
```diff
@@ -2,15 +2,14 @@ version = 2.0.0-RC5
...
 docstrings                 = JavaDoc
 indentOperator             = spray
 maxColumn                  = 120
 rewrite.rules              = [RedundantParens, SortImports, AvoidInfix]
 unindentTopLevelOperators  = true
 align.tokens               = [{code = "=>", owner = "Case"}]
-align.openParenDefnSite    = true
-align.openParenCallSite    = true
+align.openParenDefnSite    = false
```
Comment (helena) [inline .scalafmt.conf]: Redundant? Default changed from true to false in v1.6. https://scalameta.org/scalafmt/docs/configuration.html#alignopenparendefnsite

File: .scalafmt.conf [hunk 3]
```diff
@@ -2,15 +2,14 @@ version = 2.0.0-RC5
...
+align.openParenCallSite    = false
```
Comment (helena) [inline .scalafmt.conf]: Same as above

**Pull request merged**